### PR TITLE
Add test command

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #2

### CODE REVIEW

File: src/commands/test.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      type: String,
+    },
+    {
+      name: 'visibility',
+      description: "Set job visibility to 'public', 'public restricted', 'share', 'team' or 'private'",
+      type: String,
+    },
+  ];
+
+  run(options): Promise<any> {
```
Comment (samccone) [inline src/commands/test.ts]: this will either be an Error or null so we could do like Error|null right?

Comment (justinfagnani) [inline src/commands/test.ts]: The error type is not part of Promise's type parameters. I can change `any` to `void` though.

File: src/commands/test.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
+import {Command} from './command';
+import * as wct from 'web-component-tester';
+// import * as process from 'process';
+
+export class TestCommand implements Command {
+  name = 'test';
+
+  description = 'Runs web-component-tester';
+
+  args = [
```
Comment (garlicnation) [inline src/commands/test.ts]: Is there any way to get this from wct, or are we only going to try that for the newer batch of command-line-arg based tools?

**Comment by garlicnation** [comment]
LGTM

File: src/commands/test.ts (hunk 2 above)
Comment (justinfagnani) [inline src/commands/test.ts]: Yeah, this is somewhat temporary, and obviously not very maintainable. I either want to standardize on command-line-args style declarations, or add some mapping functions.

**Pull request merged**